<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAddStudentWindow</class>
 <widget class="QMainWindow" name="CAddStudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="stuNumLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>学号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stuNumEdi">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addStuBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stuNamEdi">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="stuNamLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionAddStu"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFunction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddStu">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
